airflow/providers/google/cloud/hooks/vertex_ai/pipeline_job.py

```diff
@@ -178,11 +178,17 @@ def run_pipeline_job(
         service_account: str | None = None,
         network: str | None = None,
         create_request_timeout: float | None = None,
-        experiment: str | experiment_resources.Experiment | None = None,
```

Comment:
What is the reason for removing this parameter? Also we can't remove parameters without deprecating them first.

Reply:
This parameter is no longer used in any of the underlying methods. Is there any procedure for deprecation?

Reply:
Parameter kept.